flask_common/utils.py

```diff
@@ -719,7 +719,10 @@ def get_ignored_collections(self):
 
 
     def _get_queries(self):
-        ignore_query = { "ns": {"$nin": self.get_ignored_collections()}, "op": { "$ne": "killcursors" } }
+        ignore_query = { "$or": [
+            { "ns": {"$nin": self.get_ignored_collections()}, "op": { "$ne": "killcursors" } },
+            { "ns": "{0}.$cmd".format(self.db.name), "command.findAndModify": { "$exists": True } },
```

Comment:
The variable is confusing, it should be called `include_query` or just `filter_query`. They show up as under `$cmd` so we exclude them by default from query counting.

Note that this is only relevant with the new event logging which replaces regular updates/deletes with `findAndModify`.